backend/src/test/java/ch/puzzle/pcts/service/validation/CertificateValidationServiceTest.java

```diff
@@ -6,303 +6,150 @@
-        certificate.setCertificateType(CertificateType.CERTIFICATE);
-
-        assertDoesNotThrow(() -> validationService.validateOnCreate(certificate));
-    }
-
-    @DisplayName("Should throw exception on validateOnCreate() when certificate type is not certificate")
-    @ParameterizedTest
-    @MethodSource("leadershipExperienceTypes")
-    void shouldThrowExceptionOnValidateOnCreateWhenCertificateTypeIsNotCertificate(CertificateType certificateType) {
-        Certificate certificate = new Certificate();
-        certificate.setName("Valid name");
-        certificate.setPoints(BigDecimal.valueOf(50));
-        certificate.setCertificateType(certificateType);
-
-        PCTSException exception = assertThrows(PCTSException.class,
-                                               () -> validationService.validateOnCreate(certificate));
-
-        assertEquals("Certificate type is not certificate.", exception.getReason());
-        assertEquals(ErrorKey.CERTIFICATE_TYPE_IS_NOT_CERTIFICATE, exception.getErrorKey());
-    }
-
-    @DisplayName("Should be successful on validateOnDelete() when id is valid")
-    @Test
-    void shouldBeSuccessfulOnValidateOnDeleteWhenIdIsValid() {
-        Long id = 1L;
-        when(persistenceService.getById(id)).thenReturn(Optional.of(new Certificate()));
-
-        assertDoesNotThrow(() -> validationService.validateOnDelete(id));
-    }
-
-    @DisplayName("Should throw exception on validateOnDelete() when id is invalid")
-    @Test
-    void shouldThrowExceptionOnValidateOnDeleteIdWhenIdIsInvalid() {
-        Long id = -1L;
-        when(persistenceService.getById(id)).thenReturn(Optional.empty());
-
-        PCTSException exception = assertThrows(PCTSException.class, () -> validationService.validateOnDelete(id));
-
-        assertEquals("Certificate with id: " + id + " does not exist.", exception.getReason());
-        assertEquals(ErrorKey.NOT_FOUND, exception.getErrorKey());
-    }
-
-    @DisplayName("Should be successful on validateOnUpdate() when id is valid")
-    @Test
-    void shouldBeSuccessfulOnValidateOnUpdateWhenIdIsValid() {
-        Long id = 1L;
-        Certificate certificate = new Certificate();
-        certificate.setName("Valid name");
-        certificate.setPoints(BigDecimal.valueOf(5));
-        certificate.setCertificateType(CertificateType.CERTIFICATE);
-
-        when(persistenceService.getById(id)).thenReturn(Optional.of(new Certificate()));
-
-        assertDoesNotThrow(() -> validationService.validateOnUpdate(id, certificate));
-    }
-
-    @DisplayName("Should throw exception on validateOnUpdate() when id is invalid")
-    @Test
-    void shouldThrowExceptionOnValidateOnUpdateIdWhenIdIsInvalid() {
-        Long id = -1L;
-        Certificate certificate = new Certificate();
-        certificate.setName("Valid name");
-
-        when(persistenceService.getById(id)).thenReturn(Optional.empty());
+    @Mock
+    private CertificatePersistenceService persistenceService;
```

Comment:
[MED–CG]: this is against our coding guidelines, move this before the `@InjectMock` statement.